packages/kbn-cli-dev-mode/src/get_server_watch_paths.test.ts

```diff
@@ -71,6 +71,7 @@ it('produces the right watch and ignore list', () => {
       <absolute path>/x-pack/plugins/lists/server/scripts,
       <absolute path>/x-pack/plugins/security_solution/scripts,
       <absolute path>/x-pack/plugins/security_solution/server/lib/detection_engine/scripts,
+      <absolute path>/x-pack/plugins/metrics_summary/server/scripts,
```

Comment:
Should this be `metrics_entities`?
```suggestion
      <absolute path>/x-pack/plugins/metrics_entities/server/scripts,
```